docs/README.md

```diff
@@ -1,73 +1,127 @@
 
-If you feel compelled to give further details about the block, such as it's **relationship with the VTEX admin**, don't hesitate to use this section.
+```json
+"product-summary.shelf": {
+  "children": [
+    "product-list-price",
+    "product-selling-price",
+    "product-price-savings",
+    "product-installments"
+  ]
+},
+```
 
-## Customization
+Every block in this app has some props in common:
 
-The first thing that should be present in this section is the sentence below, showing users the recipe pertaining to CSS customization in blocks:
+| Prop name          | Type      |  Description | Default value |
+| --------------------| ----------|--------------|---------------|
+| `markers`           |`string[]` | Custom markers used to customize parts of the message on Site Editor|`[]`|
+|  `blockClass`  |  `string`  |  Unique  block  ID  to  be  used  in [CSS  customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization#using-the-blockclass-property)  |  `undefined`  |
 
-`In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).`
+For example:
 
-Thereafter, you should add a single column table with the available CSS handles for that block:
+```json
+"product-price-savings#summary": {
+  "props": {
+    "markers": [
+      "discount"
+    ],
+    "blockClass": "summary"
+  }
+},
+```
 
-| CSS Handles |
-| ----------- |
-| `XXXXX`     |
-| `XXXXX`     |
-| `XXXXX`     |
-| `XXXXX`     |
-| `XXXXX`     |
+### Step 3 - Editing the values on Site Editor
+
+Every Product Price's block uses the [ICU Message Format](https://format-message.github.io/icu-message-format-for-translators/).
 
-If there are none, add the following sentence instead:
+This makes it possible to fully edit the messages exported by each Product Price's block. Check it out in the admin's Site Editor:
 
-`The component still doesn't have CSS Handles for its specific customization.`
+![image](https://user-images.githubusercontent.com/8443580/77782384-f6896b00-7035-11ea-8808-fb2a5533d1a6.png)
 
----
+Each of these messages has `markers` responsible for wrapping a part of the message with its own CSS handle. To use `markers` in your message, do the following:
 
-Check out some documentation models that are already live:
+1 - Add to the prop `markers` a marker name, for example: `"markers": ["highlight"]`
+2 - Open the Site Editor and click to edit the block
+3 - In the edition form of the block, fill the field with a text and wrap the piece of a text using the marker as if it were an element. Example: `<highlight>-{savingsPercentage}</highlight>`
+4 - Save the changes
+5 - If you inspect the HTML of the element you will notice that this part of the text is wrapped in a new `span` with its own unique selector: `<span class="vtex-product-price-1-x-savings-highlight">`
```

Comment:
```suggestion

```

Reply:
How to use the `Markers` prop should be a recipe. That is the reason why I deleted this from here. I will work on the recipe and as soon as it gets ready I will create a link to it in the prop table.